okio/src/commonMain/kotlin/okio/internal/Sha1MessageDigest.kt

```diff
@@ -13,63 +13,174 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package okio.internal
 
-internal class Sha1MessageDigest : AbstractMessageDigest() {
+@ExperimentalUnsignedTypes
+internal class Sha1MessageDigest : OkioMessageDigest {
+  private var messageLength = 0L
+  private val unprocessed = ByteArray(64)
+  private var unprocessedLimit = 0
+  private val words = IntArray(80)
+
+  private var h0: Int = 0x67452301
+  private var h1: Int = 0xEFCDAB89u.toInt()
```

Comment:
For example from the comment above...we could open a scratch file and calculate this value and replace it with the proper `Int` value. That way we could drop unsigned typed altogether. 

However, I'm not sure if that's your intention!

Reply:
Yeah, I’d like to go away from unsigned types. Rather annoyingly there’s few nice ways to do it until unsigned types become a part of the language.